### ISSUE TITLE
FEATURE REQUEST: Close all open playlists

### ISSUE THREAD

**Issue by ieee8023** [clementine-player/Clementine#5867]
I always end up having tons of open playlists. I just want to close them all and start over. Can there be an option if I right click on the playlist tab to "Close all playlists". And then a way not to save them all? Like a check box to "Don't ask again" when the prompt comes to save the playlists. 

Until then here is my solution for those wanting this option. The playlists can be cleared manually by deleting the playlists from the sqlite database. On Mac this is located in ~/Library/Application Support/Clementine"

```
sqlite3 clementine.db "DELETE FROM playlist_items;"
sqlite3 clementine.db "DELETE FROM playlists;"
```

**Comment by chrisgraham** [comment]
Agreed. There needs to be features similar to in IDEs, where you can "Close all tabs" and "Close other tabs". Then you can choose which to save, or save all (ideally on one dialog with checkboxes).
